Tests/WriteFreelyTests/TestUtils.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import XCTest
+import WriteFreely
+
+final class MockURLSessionDataTask: URLSessionDataTaskProtocol {
+    private (set) var resumeWasCalled = false
+
+    func resume() {
+        resumeWasCalled = true
+    }
+}
+
+final class MockURLSession: URLSessionProtocol {
+    var nextDataTask = MockURLSessionDataTask()
+    var nextData: Data?
+    var nextError: Error?
+    var expectedStatusCode: Int = 200
```

Comment:
This lets us set our expected status code explicitly — not everything expects a `200 OK`!